assets/dev/js/editor/container/container.js

```diff
@@ -162,6 +162,38 @@ export default class Container extends ArgsObject {
 		this.requireArgumentInstance( 'model', Backbone.Model, args );
 	}
 
+	/**
+	 * Function getRelatedControls().
+	 *
+	 * Example:
+	 * Settings = { typography_typography: 'whatever', button_text_color: 'whatever' };
+	 * Result { control_name: controlValue, ... - and so on };
+	 * `Object.keys( Result ) = [ 'typography_typography', 'typography_font_family', 'typography_font_size', 'typography_font_size_tablet', 'typography_font_size_mobile', 'typography_font_weight', 'typography_text_transform', 'typography_font_style', 'typography_text_decoration', 'typography_line_height', 'typography_line_height_tablet', 'typography_line_height_mobile', 'typography_letter_spacing', 'typography_letter_spacing_tablet', 'typography_letter_spacing_mobile', 'button_text_color' ]`.
+	 *
+	 * @param {{}} settings
+	 *
+	 * @return {{}}
+	 */
+	getRelatedControls( settings ) {
```

Comment:
Since this method is a group-related, its name should reflect it. Something like "getGroupRelatedControls"

Reply:
Handled. Force-Pushed.